client/i18n/locales/english/translations.json

```diff
@@ -66,8 +66,9 @@
   },
   "landing": {
     "big-heading-1": "Learn to code — for free.",
-    "big-heading-2": "Build projects.",
-    "big-heading-3": "Earn certifications.",
+    "big-heading-2": "Learn to solve problems.",
+    "big-heading-3": "Build projects.",
+    "big-heading-4": "Earn certifications.",
```

Comment:
This change here is adding a new key to be translated through the `reacti18next` functions...